scheduler.rst

```diff
@@ -226,6 +258,32 @@ defined by PHP datetime functions::
     $until = '2023-06-12';
     RecurringMessage::every('first Monday of next month', new Message(), $from, $until);
 
+Like cron triggers, you can also use the
+:class:`Symfony\\Component\\Scheduler\\Attribute\\AsPeriodicTask` attribute
+on an invokable class::
+
+    // src/Scheduler/Task/SendDailySalesReports.php
+    namespace App\Scheduler\Task;
+
+    use Symfony\Component\Scheduler\Attribute\AsPeriodicTask;
+
+    #[AsPeriodicTask(frequency: '1 day', from: '2022-01-01', until: '2023-06-12')]
```

Comment:
Should we talk about ? 

`frequency` can also be an int (from my understand it's for seconds)
`from` can be nullable
`until` can be nullable
`jitter`
`schedule`

Reply:
Added some other as well!